docs/access.rst

```diff
@@ -49,6 +49,8 @@ Starting Kolibri on Linux and MacOS will differ depending on the method you used
 
     kolibri start
 
+  .. warning:: On macOS you may need to prefix the command with ``python``, and type ``python kolibri start`` instead.
```

Comment:
I'm not sure that works and I have no Mac to test it. However, `python -m kolibri start` will work.

Reply:
Note was requested by the @imps team, as it happened to one of them when trying to start the PEX on her computer.

cc @indirectlylit

Reply:
> Note was requested by the @imps team, as it happened to one of them when trying to start the PEX on her computer.

Above, you have a separate bullet for "If you used the PEX package..."

In the case of PEX, users need to make sure they:

```bash
chmod +x kolibri-v0.10.0.pex
./kolibri kolibri-v0.10.0.pex start  # note './'
```

The advice given might work, but it's confusing to me that in the case of a PEX there would be something called `kolibri` accessible on the path.

Reply:
(also note this advice appears to be under the section for PPA and Pip)

Reply:
@indirectlylit it's valid if someone installs on Mac via `pip install kolibri`, that's the only case I can think of. But I still think it would have to be with the `-m` switch to call the `kolibri` _module_ in cases where the command doesn't enter the PATH environment.